manifestival.go

```diff
@@ -30,13 +33,19 @@ type Manifestival interface {
 	Transform(fns ...Transformer) error
 }
 
+// Manifest tracks a set of concrete resources which should be managed as a
+// group using a Kubernetes client provided by `NewManifest`.
 type Manifest struct {
 	Resources []unstructured.Unstructured
 	client    client.Client
 }
 
 var _ Manifestival = &Manifest{}
 
+// NewManifest creates a Manifest from a comma-separated set of yaml files
+// (or directories if the `recursive` option is set). The Manifest will be
```

Comment:
```suggestion
// or directories (and subdirs if `recursive` is true). The Manifest will be
```

Reply:
Thanks, I'd glanced over `Parse` too quickly.

Reply:
(Updated, force-pushed to have only a single commit)